### DISCUSSION TITLE
[firebase_ml_vision] - Face detection and video recording running in parallel

### DISCUSSION THREAD

**Discussion by kroikie** [firebase/flutterfire#5554] [Q&A]
Hi, I was wondering if there is any way to use the ML kit (face detection) in real-time while also running the video recording in parallel? After inspecting the camera module, it seems that it can only do one or the other, as ML kit takes a CameraImage argument inside .startImageStream() and a String path argument inside .startVideoRecording(). 

It would be helpful if it's possible to do both at the same time. I've been exploring other methods of recording the stream as well, via encoders such as FFmpeg to resolve the conflict of camera controllers but it suffered from performance issues. Any form of assistance would be deeply appreciated.

**Comment by kroikie** [comment]
@ysrarff

The issue at https://github.com/flutter/flutter/issues/35564 has been closed and moved here. Future collaboration on this issue will be done here.

**Comment by jtkeyva** [comment]
@kroikie @BondarenkoStas @iapicca @Ehesp @PolinaGo any luck with this?
thanks!
